src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,157 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
+    private static class Node<T> {
+        private Node<T> left;
+        private Node<T> right;
```

Comment:
```suggestion
        private Node<T> prev;
        private Node<T> next;
```

lets use classic naming?